app.js

```diff
@@ -10,6 +10,11 @@
 var express = require('express');
 var db = require('./db');
 var app = express();
+var adapterManager = require('./adapter-manager.js');
+var bodyParser = require('body-parser');
```

Comment:
OK, I was just curious. If you think it's worth adding a dependency then let's keep it.